packages/codegen/src/EntityDbMetadata.ts

```diff
@@ -241,6 +245,10 @@ export class EntityDbMetadata {
 
     this.tableName = table.name;
     this.tagName = config.entities[this.entity.name]?.tag;
+
+    const { createdAtConf, updatedAtConf } = getTimestampConfig(config);
+    this.createdAt = this.primitives.find((f) => createdAtConf.names.includes(f.columnName));
+    this.updatedAt = this.primitives.find((f) => updatedAtConf.names.includes(f.columnName));
```

Comment:
Baking into the metadata what, if any, the created at / updated at field names are for each entity.

In our app, these will all be the same; I toyed with detecting "literally all of the entities use the same updated at / created at" and then "putting that config in just one spot", instead of putting it in every metadata instance, but, dunno, I couldn't find a great spot for it.

And this doesn't seem that bad.